conf/defaults.ini

```
@@ -1079,6 +1079,17 @@ max_attempts = 3
 # The interval string is a possibly signed sequence of decimal numbers, followed by a unit suffix (ms, s, m, h, d), e.g. 30s or 1m.
 min_interval = 10s
 
+# Disable the internal alertmanager and only use external ones. The default value is `false`.
+disable_internal_alertmanager = true
+
+# URL of the Alertmanager to replace the internal one.
+main_alertmanager_url = http://localhost:8006
+
+# Credentials for the external Alertmanager.
+main_alertmanager_basic_auth_user =
+
+main_alertmanager_basic_auth_password =
```

```suggestion
main_alertmanager_basic_auth_password =

tenant_mappings = orgID tenantID
```

e.g. `tenant_mappings = 1 123456`